sdk/spring/spring-cloud-azure-core/src/main/java/com/azure/spring/cloud/core/implementation/util/AzurePropertiesUtils.java

```diff
@@ -99,10 +99,20 @@ public static <T extends AzureProperties> void mergeAzureCommonProperties(AzureP
      * @param source The source object.
      * @param target The target object.
      */
-    public static void copyPropertiesIgnoreNull(Object source, Object target) {
+    public static void copyPropertiesIgnoreSourceNull(Object source, Object target) {
         BeanUtils.copyProperties(source, target, findNullPropertyNames(source));
     }
 
+    /**
+     * Copy properties from source object to target object. Ignore the source value if the related target is non-null.
+     *
+     * @param source The source object.
+     * @param target The target object.
+     */
+    public static void copyPropertiesIgnoreTargetNonNull(Object source, Object target) {
```

Comment:
Why not directly use:
BeanUtils.copyProperties(source, target, findPropertyName(xxx));

seems it not add meaning or reduce some codes?